fex2/preconfigured/experiments/splash/plot.py

```diff
@@ -59,23 +63,27 @@ def build_plot(infile: str, outfile: str, plot_type: str = 'speedup'):
     df = DataFrame(pivoted.to_records()).set_index("benchmark", drop=True)
 
     # rename builds
-    df.rename(columns=RENAMINGS, inplace=True)
+    df.rename(columns=RENAMINGS_SPEEDUP, inplace=True)
 
     # the resulting table
     helpers.debug("Plot data\n" + str(df))
 
     # build the plot
     helpers.debug("Building a plot")
     plt = plot.BarPlot(style=SplashBarPlotStyle)
-    plt.build(df,
-              title="Speedup of GCC -O3 optimizations",
-              ylabel="Normalized runtime\n(w.r.t. native GCC)",
-              vline_position=11.5
-              )
+    plt.build(
+        df,
+        title="Speedup of GCC -O3 optimizations",
+        ylabel="Normalized runtime\n(w.r.t. native GCC)",
+        vline_position=11.5
+    )
 
     plt.savefig(
         outfile,
         dpi="figure",
         pad_inches=0.1,
         bbox_inches='tight'
     )
+
+
+build_plot("/home/kairaedsch/Documents/git-privat/fex/fex2/preconfigured/splash-collected.csv", "/home/kairaedsch/Documents/git-privat/fex/fex2/preconfigured/splash.pdf")
```

Comment:
path

Reply:
Oh, that's quite embarrassing :see_no_evil: